src/sentry/static/sentry/app/api.jsx

```diff
@@ -191,13 +191,12 @@ export class Client {
     return this.activeRequests[id];
   }
 
-  requestPromise(path, options = {}) {
+  requestPromise(path, {includeAllArgs, ...options} = {}) {
     return new Promise((resolve, reject) => {
       this.request(path, {
         ...options,
-        success: (data, ...args) => {
-          // This fails if we need jqXhr :(
-          resolve(data);
+        success: (data, successString, allArgs) => {
+          includeAllArgs ? resolve([data, allArgs, successString]) : resolve(data);
```

Comment:
We should resolve in the same order
```suggestion
          includeAllArgs ? resolve([data, successString, allArgs]) : resolve(data);
```